README.md

```diff
@@ -1 +1,180 @@
-# math-magician
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <!-- LOGO -->
+
+  <img src="/public/logo512.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>React App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 React App <a name="about-project"></a>
+
```

Comment:
- [ ] As required by MIcroverse, your readme file needs to be descriptive and look professional, I'll suggest you remove this react logo and also name your project properly. Also, this is a react app, but the project name is `Math Magician`. :+1: :+1: 

- - Screenshot:
![Selection_006](https://github.com/anyars-encarta/math-magician/assets/107632780/7c032aea-0c70-49dc-bb6f-5009e1e34225)

Reply:
I will correct that